### DISCUSSION TITLE
Blank window

### DISCUSSION THREAD

**Discussion by AldahirGut** [apache/netbeans#3693] [Q&A]
Hello, I have a problem, when I open netbeans it appears the little loading window, but when it opens, it stays in a white blank window, I've been reinstalling for all this week, jdk, netbeans and dependencies (I'm in linux), I was using it last week with no problem, but I changed the distro and after that it started to give me that problem. I would apreciate if someone can help me 'cause I need it for the school.
![Screenshot_2022-03-04-10-56-53_1366x768](https://user-images.githubusercontent.com/100958407/156809085-32f2dc34-b489-4b8a-8151-5f184237b47a.png)
![Screenshot_2022-03-04-10-57-05_1366x768](https://user-images.githubusercontent.com/100958407/156809104-d5d6e935-7b5c-4033-96b1-2318fd1e8d2e.png)

**Comment by geertjanw** [comment]
Try the latest release, released today, see netbeans.apache.org.

**Reply by AldahirGut** [reply]
I've tried to install it and it doesn't work anyway

**Comment by mbien** [comment]
to which distro / window manager did you switch? If it uses Wayland you probably will have to run it on XWayland.

**Reply by AldahirGut** [reply]
I'm on Archraft with bspwm, I used Archraft with bspwm before and it worked perfectly, but now it doesn't

**Reply by mbien** [reply]
bspwm doesn't use wayland according to their wiki, so using xwayland for backward compatibility wouldn't do anything anyway.

I suspect this won't be a NetBeans specific issue, quick search:
https://wiki.archlinux.org/title/Bspwm (skip to the java section)
https://shroudri.github.io/guides/fixing-java-and-bspwm/
https://bbs.archlinux.org/viewtopic.php?id=181865
etc
good luck!

**Reply by AldahirGut** [reply]
thank you, I will check it out
